benchmark/blob/blob-text.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+const common = require('../common.js');
+const { Blob } = require('buffer');
+
+const bench = common.createBenchmark(main, {
+  bytes: [0, 8, 128],
```

Comment:
128 bytes seems rather small. I guess this is mainly used for bigger values such as 128kb, 1mb and similar.